src/python/TaskWorker/Actions/DagmanCreator.py

```diff
@@ -1204,11 +1214,12 @@ def executeInternal(self, *args, **kw):
             sandboxName = kw['task']['tm_user_sandbox']
             dbgFilesName = kw['task']['tm_debug_files']
             try:
-                downloadFromS3(crabserver=self.crabserver, objecttype='sandbox', username=username,
-                               tarballname=sandboxName, filepath=sandboxTarBall, logger=self.logger)
+                self.logger.debug(f"Checking if sandbox file is available: {sandboxName}")
```

Comment:
comment to myself: let's add a comment explaining why we have this check. Yet it seems an overkill, I do not remember any situation where crab submit succeeded and sandbox was not available.

Reply:
something like "very unlikely that it ever happens, but better to find out here than in the scheduler when bootstrapping the DAG"